### ISSUE TITLE
CSV Reader not working

### ISSUE THREAD

**Issue by heinzwilli** [bouviervj/topojson-j#5]
Hi,
in your commit "Better management for color management, update for csv management" you changed the CSVReader. In src/json/converter/csv/CSVReader.java function readRecord() you changed
```java
boolean filter = (prop!=null?applyFilter(aRecordNumber,prop):true); 
```
to
```java
boolean filter = (prop!=null?applyFilter(aRecordNumber,prop):false);
```
this caused the function
```java
TopojsonApi.shpToTopojsonFile("./data/MA.shp", "nad83:2001",
					"./web/topojson_10.json", 
					"MA", 
					10, 
					4, 
					false);
```
to skip reading all the records.

**Comment by bouviervj** [comment]
Yes, sorry about that , if I recall well , you should have a dbf file associated with the shp file that maps record # to data. This is what defines prop field. In fact the code you highlight is in ShpFileReader.java . I have set the value to false to optimize the parsing of big shop files. As you can merge associated data (dbf content) with cvs files, you can have a subset of record # to filter shape records from the shape file. 
This can be used to filter element that have no prop on large files.
Let me know if your usage of this method relies only on shop file, meaning you'll never have the dbf attached file, so that I will try to figure out how to bypass this in defining this boolean.

**Comment by heinzwilli** [comment]
I only had a .shp file and no .dbf file.

**Comment by bouviervj** [comment]
[Should be solved with this commit](https://github.com/bouviervj/topojson-j/commit/648a20fd2799013a1c968c5f378ea987f3301c46) . let me know
